charts/jenkins/templates/_helpers.tpl

```diff
@@ -109,6 +109,9 @@ jenkins:
     enabled: true
   mode: {{ .Values.controller.executorMode }}
   numExecutors: {{ .Values.controller.numExecutors }}
+  {{- if not (kindIs "invalid" .Values.controller.customJenkinsLabels) }}
```

Comment:
@timja do you prefer a null check here or should we simplify it to

if .Values.controller.customJenkinsLabels

Reply:
Hmm will empty string work with the second? As you probably want to be able to set an empty string to clear them

Reply:
Your suspicions seem to be correct, @timja. Tested using both the current condition and `if .Values.controller.customJenkinsLabels` with this as the values file:
```
controller:
  customJenkinsLabels: ""
```
It works as desired when we use the null check, but not with the simple if statement